CHANGELOG.rst

```diff
@@ -2,6 +2,94 @@
+
+**New features**
+
+* [:backend:`4606`] Improved ZGW registrations options and made it consistent with Objects API.
+    - React-Select is now used for dropdowns.
+    - Catalogue can be specifically selected (same UI as Objects API).
+    - Case type can be selected by identification.
+    - Document types can now be filtered down to particular case type.
+    - Updated more Dutch to English code in API endpoints.
+    - Added more tests.
+    - Added validation to document type reference.
+    - Added the serializer option to allow selecting document type by description (omschrijving).
+    - Deprecated ZGW url-based document type configuration.
+* [:backend:`4431`] Improved addressNL mapping backwards compatibility.
+* [:backend:`4418`] Made it possible to map specific addressNL fields or a whole object to ObjectsAPI.
+* [:backend:`4693`] Integrated Objects API prefill modal with backend.
+    - Added copy button to the modal, which allows the user to copy the configuration used by the Form's
+      registration backend. This is shown only when backends are available.
+    - Added dropdown to select a registration backend instead of always picking the first one.
+* [:backend:`4745`] Added registration variable containing all attachment URLs.
+* [:backend:`4796`] Added product url to ZGW APIs registration.
+* [:backend:`4762`] Added cosign data (bsn) to the StUF-ZDS creerzaak bericht.
+* [:backend:`4798`] Added custom confirmation modal, which replaces the old way of displaying confirmation
+  messages.
+* [:backend:`4344`] It's now possible to configure Objects API group in ZGW registration. This used to be by
+  default the one with the lowest primary key.
+* [:backend:`4320`] Cosign improvements. 
+    - Updated translations so the suggested texts are already included.
+    - Added the ability to add a 'cosign now' button on confirmation page.
+    - Made the cosign verification flow faster when the code is included in the link in the email.
+    - Made the confirmation page title and body configurable.
+    - Updated cosign configuration docs and manual.
+* [:backend:`4815`] Changed submission removal limit to 0, allowing submissions to be deleted after 0 days
+  (i.e. on the same day).
+
+* [:backend:`4717`] Improved accessibility for site logo, error message element and PDF documents. 
+* Replaced duplicated code for payment/registration plugin configuration option forms, by adding a generic
+  component.
+* Refactored demo plugins configuration to use a modal instead of inline configuration options.
+* [:backend:`4707`] Added the option to resize the json widget manually.    
+* [:backend:`4686`] Improved UX for email and StUF-ZDS registrations by using React components, instead of
+  react-json-schema-form generated forms.
+* [:backend:`4720`] Improved accessibility for the skiplink and the PDF report.
+* [:backend:`4396`] Added Objects API prefill plugin.
+
+**Bugfixes**
+
+* Fixed tests after removing password component.
```

Comment:
```suggestion
```
really belongs together with the change that the password component is removed tbh